config/locales/account_reset/en.yml

```diff
@@ -7,9 +7,9 @@ en:
       cancel_button: Cancel delete account
       title: Cancel delete account
     confirm_delete_account:
-      cta: You may %{link} or close this window if you're done.
-      info: The account for <strong>%{email}</strong> has been deleted. We sent an
-        email confirmation of the account deletion.
+      cta_html: You may %{link} or close this window if you're done.
```

Comment:
Do we need to make this same change in Spanish and French translations?

Reply:
OMG yes...